texas-hearing-institute/package.json

```diff
@@ -1,46 +1,37 @@
 {
-	"name": "texas-hearing-institute",
-	"version": "1.0.0",
-	"main": "node_modules/expo/AppEntry.js",
-	"scripts": {
-		"start": "expo start",
-		"android": "expo start --android",
-		"ios": "expo start --ios",
-		"web": "expo start --web",
-		"lint": "eslint . --ext .ts",
-		"prettier-format": "prettier . --write"
-	},
-	"dependencies": {
-		"@react-native-async-storage/async-storage": "^1.19.3",
-		"@rneui/base": "^4.0.0-rc.7",
-		"@rneui/themed": "^4.0.0-rc.8",
-		"@supabase/supabase-js": "^2.38.2",
-		"expo": "~49.0.13",
-		"expo-status-bar": "~1.6.0",
-		"react": "18.2.0",
-		"react-native": "0.72.5",
-		"react-native-config": "^1.5.1",
-		"react-native-dotenv": "^3.4.9",
-		"react-native-elements": "^3.4.3",
-		"react-native-url-polyfill": "^2.0.0",
-		"tailwind-react-native-classnames": "^1.5.1",
-		"tailwind-rn": "^4.2.0"
-	},
-	"devDependencies": {
-		"@babel/core": "^7.20.0",
-		"@types/react": "~18.2.14",
-		"@typescript-eslint/eslint-plugin": "^6.9.0",
-		"@typescript-eslint/parser": "^6.9.0",
-		"eslint": "^8.52.0",
-		"husky": "^8.0.3",
-		"prettier": "^3.0.3",
-		"tailwindcss": "^3.3.3",
-		"typescript": "^5.2.2"
-	},
-	"private": true,
-	"husky": {
```

Comment:
oops, don't overwrite these changes for package.json since it includes the linting + other stuff. i think just start with the deleted changes + add in any new packages y'all installed (like ui-kitten)